web/src/favicon.ts

```diff
@@ -51,8 +51,8 @@ export function update_favicon(new_message_count: number, pm_count: number): voi
             new_message_count === 0
                 ? ""
                 : pow < 5
-                ? `${Math.floor(new_message_count / 1e3 ** pow)}${suffix}`
-                : "∞";
+                  ? `${Math.floor(new_message_count / 1e3 ** pow)}${suffix}`
+                  : "∞";
```

Comment:
If you’re curious: https://prettier.io/blog/2023/11/13/3.1.0#add-indentation-back-to-nested-ternaries-9559httpsgithubcomprettierprettierpull9559-by-rattrayalexhttpsgithubcomrattrayalex

Reply:
I can't say that I like the outcome in `sentry.ts`, but I blame misuse of ternary operations (that one should just be a function).